pkg/integrations/redis_exporter/redis_exporter.go

```diff
@@ -41,6 +41,7 @@ type Config struct {
 	RedisUser               string             `yaml:"redis_user,omitempty"`
 	RedisPassword           config_util.Secret `yaml:"redis_password,omitempty"`
 	RedisPasswordFile       string             `yaml:"redis_password_file,omitempty"`
+	RedisPasswordMapFile    string             `yaml:"redis_password_map_file,omitempty"`
```

Comment:
WDYT about making it be invalid to set more than one of these at a time?

Reply:
I think that would make things clearer, I had just been deferring to the downstream